packages/flutter_driver/lib/src/driver/driver.dart

```diff
@@ -200,6 +200,8 @@ abstract class FlutterDriver {
     await sendCommand(WaitFor(finder, timeout: timeout));
   }
 
+  /// For more details, See also [FlutterDriver.runUnsynchronized]
```

Comment:
This should be moved to the end and be more like
https://github.com/flutter/flutter/blob/a51dc3d913344f3073da12f4aae654c3299dec6c/packages/flutter/lib/src/widgets/framework.dart#L704-L706
to better fit the style of other see also sections.

Reply:
done